packages/amplify-cli/src/commands/env/import.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+module.exports = {
+  name: 'import',
+  run: async (context) => {
+    const envName = context.parameters.options.name;
+    if (!envName) {
+      context.print.error('You must pass in the name of the environment using the --name flag');
+      process.exit(1);
+    }
+
+    let config;
+    try {
+      config = JSON.parse(context.parameters.options.config);
+    } catch (e) {
+      context.print.error('You must pass in the configs of the environment in an object format using the --config flag');
+      process.exit(1);
+    }
+
+    let awsInfo;
+    if (context.parameters.options.awsInfo) {
+      try {
+        awsInfo = JSON.parse(context.parameters.options.awsInfo);
+      } catch (e) {
+        context.print.error('You must pass in the AWS credential info in an object format for intializating your environment using the --awsInfo flag');
+        process.exit(1);
+      }
+    }
+
+    let envFound = false;
+    const allEnvs = context.amplify.getEnvDetails();
+    Object.keys(allEnvs).forEach((env) => {
```

Comment:
Nit: instead of using `forEach` you could use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) to do the search